addons/openebs/3.3.0/install.sh

```diff
@@ -48,12 +48,58 @@ function openebs() {
 function openebs_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
+            # show validation errors from pvmigrate
+            # if there are errors, maybe_prompt_migrate_from_rook_dryrun_errors() will bail
+            maybe_prompt_migrate_from_rook_dryrun_errors
             rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS"
             DID_MIGRATE_ROOK_PVCS=1 # used to automatically delete rook-ceph if object store data was also migrated
         fi
     fi
 }
 
+function maybe_prompt_migrate_from_rook_dryrun_errors() {
```

Comment:
WDYT about something like rook_to_openebs_migration_checks only?

Reply:
The `maybe_` prefix for naming functions is quite prevalent for functions which are contingent up performing some action in this project. I do agree this func needs to a better name, however.